GoogleForCreators/web-stories-wp
# Dashboard: Applied design updates to Wellbeing template

## Summary

This PR applies the new design updates to the Wellbeing template.

## User-facing changes

Layout and text sizes have been update on the Wellbeing template based off the recommendations found in this [figma design doc](https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=766%3A61).

## Todo
- I couldn't figure out a way to import the outline blob and the editor currently doesn't support outlining shapes so I couldn't place the outlines anywhere.
- I also couldn't figure out a way to import the long narrow arrow SVG either, so I'm gonna get some help to get it imported in another PR.

## Testing Instructions

1.) Go to "Explore Templates".
2.) Hover over the Wellbeing template and click on "View".
3.) Take a look at each template page in the detail view for Wellbeing.  Make sure images are showing up and the layout appears as shown in the figma updates.

---

Fixes #2477  

## Screenshots from Editor

<img src="https://user-images.githubusercontent.com/40646372/85478173-9a6caf00-b570-11ea-97d1-da3fedc0e6d2.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85349897-83688700-b4b4-11ea-9c17-b6853ab697ef.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85349913-8e231c00-b4b4-11ea-928e-dc24e230e0df.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85349931-9b400b00-b4b4-11ea-99a5-ae2d03630f77.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85349942-a4c97300-b4b4-11ea-84c5-2b7a4373c011.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85349958-ae52db00-b4b4-11ea-988a-bead23001082.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85349974-b6ab1600-b4b4-11ea-9886-7cb4199805bc.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85349985-bdd22400-b4b4-11ea-83d3-36704862ee5a.png" height="200"><img src="https://user-images.githubusercontent.com/40646372/85478252-c0924f00-b570-11ea-8827-f949e107befb.png" height="200">

<img src="https://user-images.githubusercontent.com/40646372/85350008-cc204000-b4b4-11ea-8008-ef000603ce42.png" height="200">
